src/pages/about-us.astro

```diff
@@ -0,0 +1,73 @@
+---
+import FAQCard from "../components/FAQCard.astro";
+import { faqAnswer } from "../data/faq";
+import Layout from "../layouts/Layout.astro";
+---
+
+<Layout title="About Us">
+    <div class="w-full flex items-center text-left">
```

Comment:
We are getting a lot of low-level markup in our page-level components. Not necessarily for this PR, but I wonder if we can do a better job of composing UI components to make the pages a bit more readable. I may have some time this week to try to figure that out